### ISSUE TITLE
BFQ 0007 patch needs minor edit (Linux410)

### ISSUE THREAD

**Issue by jbMacAZ** [manjaro/packages-core#69]
Source has new header "blk-wbt.h"
So the last three lines of 0007-reverse-include-order-between-blk.h-and-bfq.h.patch need to be changed to
```
 #include "blk-wbt.h"
 
 /* Expiration time of sync (0) and async (1) requests, in jiffies. */
```

**Comment by philmmanjaro** [comment]
I didn't compile it yet with given patch. Also on the **linux411** branch it is the [same](https://github.com/linusw/linux-bfq/commit/a1e78683b42b02b2dc69f349ea922c28f3a4df81).

**Comment by jbMacAZ** [comment]
Linux410 builds and boots with the edit.  4.10.6 and 4.10.7 had some USB issues, which seem resolved in 4.10.8.  I haven't tried linux411, waiting for -rc5.

**Comment by philmmanjaro** [comment]
https://github.com/manjaro/packages-core/commit/fdff4487446b59322a5fcd55cf5d9d2ca96913d6
